spec.bs

```diff
@@ -196,7 +196,7 @@ The <dfn for="interest group">estimated size</dfn> of an [=interest group=] |ig|
   [=interest group/priority vector=]:

```

Comment:
maybe insert a step on line 179 that says:
```
1. Append the current [=date=] to |interestGroup|'s [=interest group/join counts=].
1. If the browser is currently storing an interest group with `owner` and `name` that matches |interestGroup|, append that interest group's [=interest group/join counts=] to |interestGroup|'s [=interest group/join counts=].
```

Reply:
Done